### DISCUSSION TITLE
Little steering while taxiing. Am I missing something?

### DISCUSSION THREAD

**Discussion by Villason** [flybywiresim/aircraft#5836] [Help]
The steering angle is quite reduced unless I am almost stopped on ground. Am I missing some kind of button press or something?

**Comment by nsMiro** [comment]
I noticed that too and also something happened with the speed accelerating and braking very quickly

**Comment by MStallen** [comment]
Same problem

**Reply by CptBayer** [reply]
me too ... same same ...
